tests/suites/os/suite.js

```diff
@@ -100,11 +101,16 @@ module.exports = {
 	},
 	tests: [
 		'./tests/fingerprint',
+		'./tests/os-release',
+		'./tests/chrony',
+		'./tests/kernel-overlap',
+		'./tests/bluetooth',
+		'./tests/healthcheck',
+		'./tests/variables',
 		'./tests/led',
```

Comment:
I thought the LED tests didn't work out?

Reply:
Fixed the LED test!